evcxr/src/command_context.rs

```diff
@@ -225,27 +226,14 @@ Panic detected. Here's some useful information if you're filing a bug report.
         Ok(completions)
     }
 
-    fn load_config(&mut self, quiet: bool) -> Result<EvalOutputs, Error> {
+    fn load_config(&mut self, _quiet: bool) -> Result<EvalOutputs, Error> {
```

Comment:
Might as well remove the quiet argument if it's no longer used. Although I guess that possibly means that the --quiet flag now does nothing?

Reply:
I think I should make it work the same as before, that is, to print the location of the files.